packages/frontend/app/(dashboard)/course/[cid]/queue/[qid]/components/modals/EditQueueModal.tsx

```diff
@@ -679,11 +680,25 @@ const EditQueueModal: React.FC<EditQueueModalProps> = ({
           </Form.List>
         </>
       )}
-      <Form.Item label="Zoom/Teams Link" name="zoomLink">
+      <Form.Item
+        label={
+          <div className="flex flex-row items-center gap-1">
+            Zoom/Teams Link
+            <Tooltip
+              title={
+                'This is the queue-specific link. If not set, the link shown to students will default to the course-wide link.'
+              }
+            >
+              <QuestionCircleOutlined style={{ color: 'gray' }} />
+            </Tooltip>
+          </div>
+        }
+        name="zoomLink"
+      >
         <Input
           allowClear={true}
           className="text-sky-800"
-          placeholder="[No Zoom/Teams link set]"
+          placeholder="[No queue-specific Zoom/Teams link set]"
```

Comment:
I wonder if there is a good way of showing the course zoom link here. What if the placeholder was changed to be the course zoom link? And if there is no course zoom link, it just says [No Zoom/Teams link set] like before

Reply:
Thats a good idea. I'll switch this over to that instead.